nugenesis9_schema_verify.sql

```diff
@@ -158,8 +160,12 @@ END;
 BEGIN
 	SELECT NGKEYVALUE INTO :V_SDMSSchemaVer FROM NGSYSUSER.NGCONFIG WHERE NGKEYID = 'BUILDNUMBER';
 	DBMS_OUTPUT.PUT_LINE('SDMS schema version: '||:V_SDMSSchemaVer);
-
+	:v_SDMSSchemaVerAsNum := 8000 --Adding logic to address issue #2. Necessary locations in the schema

```

Comment:
Need to remove the ':v_SDMSSchemaVerAsNum := 8000'
```suggestion
	--Adding logic to address issue #2. Necessary locations in the schema
```

Reply:
Hey Nick, I was thinking of having it default to 8000. Do you think it would be better to leave it uninitialized?

Reply:
> Need to remove the ':v_SDMSSchemaVerAsNum := 8000'

Hey Nick, I was thinking of having it default to 8000. Do you think it would be better to leave it uninitialized?

Reply:
Ah, got it, in that case you just need the trailing semicolon after ":v_SDMSSchemaVerAsNum := 8000"